### PR TITLE
Update README.md: Cert-Manager

### PR DESCRIPTION
**Pull request by pbajjuri20**
Please update these commands to find the YAML files in the folder.

Please verify this command, it is giving the error

```
helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr -f deploy/examples/cert-manager/istio-csr-helm-values.yaml

Error: template: cert-manager-istio-csr/templates/serviceaccount.yaml:3:7: executing "cert-manager-istio-csr/templates/serviceaccount.yaml" at <ne .Values.image.pullSecretName "">: error calling ne: incompatible types for comparison
```

### CODE REVIEW

File: deploy/examples/cert-manager/README.md [hunk 1]
```diff
@@ -16,11 +16,11 @@
 
 2. Provision certificates:
 
-        kubectl apply -f selfsigned-ca.yaml -n istio-system
+        kubectl apply -f deploy/examples/cert-manager/selfsigned-ca.yaml -n istio-system
 
 3. Install cert-manager istio-csr Service:
 
-        helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr -f deploy-prototype/istio-csr-helm-values.yaml"
+        helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr -f deploy-prototype/istio-csr-helm-values.yaml
```
Comment (rcernich) [inline deploy/examples/cert-manager/README.md:23]: This path doesn't look correct.

I think we need to decide from where users should be invoking these commands.  If they are from the same directory as this readme, they should all be `./`.  If they are from the project root, then `./deploy/examples/cert-manager/` would be correct.

Comment (pbajjuri20) [reply deploy/examples/cert-manager/README.md:23]: deploy/examples/cert-manager/

It looks good, we can user anywhere with the correct path.

Comment (pbajjuri20) [inline deploy/examples/cert-manager/README.md:23]: Can you some explanation about this step:

When I execute it has this error:

`Error: template: cert-manager-istio-csr/templates/serviceaccount.yaml:3:7: executing "cert-manager-istio-csr/templates/serviceaccount.yaml" at <ne .Values.image.pullSecretName "">: error calling ne: incompatible types for comparison`

**Comment by pbajjuri20** [comment]
@rcernich :Cert- Manager project has some issue with this step:

 

https://cert-manager.io/docs/tutorials/istio-csr/istio-csr/#installing-istio-csr

helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr \
> --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
> --set "volumeMounts[0].name=root-ca" \
> --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
> --set "volumes[0].name=root-ca" \
> --set "volumes[0].secret.secretName=istio-root-ca"
Error: template: cert-manager-istio-csr/templates/serviceaccount.yaml:3:7: executing "cert-manager-istio-csr/templates/serviceaccount.yaml" at <ne .Values.image.pullSecretName "">: error calling ne: incompatible types for comparison

**Comment by jewertow** [comment]
I'm closing this PR, because the documentation was fixed as part of #1101.

**Pull request closed without merging**